### DISCUSSION TITLE
"In [CMD-LINE]:1: Error opening configuration file"

### DISCUSSION THREAD

**Discussion by cruncher007** [haugene/docker-transmission-openvpn#2155] [Q&A] [answered]
Can't figure out what I am doing wrong. Using custom .ovpn file from xvpn.io. It updates some parameters inside the file which means it is accessible,  but then quits with the "error opening configuration file". Latest image. 

Tried playing with UID/GID, no luck, don't think it is related. OVPN file is with cert-based auth, so tried removing user name and password variables, still nothing.

Edit: Other OpenVPN clients work fine with this file.

Error log:

Starting container with revision: 20877f1b168b6ff27fc58aeef40756572c562d47
Creating TUN device /dev/net/tun
Using OpenVPN provider: CUSTOM
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for CUSTOM. Defaulting to external config
Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.tuGE0tZTbo
Extracting configs to /tmp/tmp.aRkzOUKbOk
ERROR: Could not find any configs for provider CUSTOM in downloaded configs
Cleanup: deleting /tmp/tmp.tuGE0tZTbo and /tmp/tmp.aRkzOUKbOk
Starting OpenVPN using config gb.ovpn
Modifying /etc/openvpn/custom/gb.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
Modification: Change ca certificate path
Modification: Change ping options
Modification: Update/set resolv-retry to 15 seconds
Modification: Change tls-crypt keyfile path
Modification: Set output verbosity to 3
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
Setting OpenVPN credentials...
Options error: In [CMD-LINE]:1: Error opening configuration file: /etc/openvpn/custom/gb.ovpn
Use --help for more information.

Config:

{
   "CapAdd" : [],
   "CapDrop" : [],
   "cmd" : "dumb-init /etc/openvpn/start.sh",
   "cpu_priority" : 50,
   "enable_publish_all_ports" : false,
   "enable_restart_policy" : false,
   "enabled" : true,
   "env_variables" : [
      {
         "key" : "OPENVPN_CONFIG",
         "value" : "gb"
      },
      {
         "key" : "PATH",
         "value" : "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
      },
      {
         "key" : "OPENVPN_USERNAME",
         "value" : "**redacted**"
      },
      {
         "key" : "OPENVPN_PASSWORD",
         "value" : "**redacted**"
      },
      {
         "key" : "OPENVPN_PROVIDER",
         "value" : "custom"
      },
      {
         "key" : "GLOBAL_APPLY_PERMISSIONS",
         "value" : "true"
      },
      {
         "key" : "TRANSMISSION_HOME",
         "value" : "/data/transmission-home"
      },
      {
         "key" : "TRANSMISSION_RPC_PORT",
         "value" : "9091"
      },
      {
         "key" : "TRANSMISSION_DOWNLOAD_DIR",
         "value" : "/data/completed"
      },
      {
         "key" : "TRANSMISSION_INCOMPLETE_DIR",
         "value" : "/data/incomplete"
      },
      {
         "key" : "TRANSMISSION_WATCH_DIR",
         "value" : "/data/watch"
      },
      {
         "key" : "CREATE_TUN_DEVICE",
         "value" : "true"
      },
      {
         "key" : "ENABLE_UFW",
         "value" : "false"
      },
      {
         "key" : "UFW_ALLOW_GW_NET",
         "value" : "false"
      },
      {
         "key" : "UFW_EXTRA_PORTS",
         "value" : ""
      },
      {
         "key" : "UFW_DISABLE_IPTABLES_REJECT",
         "value" : "false"
      },
      {
         "key" : "PUID",
         "value" : "1026"
      },
      {
         "key" : "PGID",
         "value" : "100"
      },
      {
         "key" : "PEER_DNS",
         "value" : "true"
      },
      {
         "key" : "PEER_DNS_PIN_ROUTES",
         "value" : "true"
      },
      {
         "key" : "DROP_DEFAULT_ROUTE",
         "value" : ""
      },
      {
         "key" : "WEBPROXY_ENABLED",
         "value" : "false"
      },
      {
         "key" : "WEBPROXY_PORT",
         "value" : "8118"
      },
      {
         "key" : "WEBPROXY_USERNAME",
         "value" : ""
      },
      {
         "key" : "WEBPROXY_PASSWORD",
         "value" : ""
      },
      {
         "key" : "LOG_TO_STDOUT",
         "value" : "false"
      },
      {
         "key" : "HEALTH_CHECK_HOST",
         "value" : "google.com"
      },
      {
         "key" : "SELFHEAL",
         "value" : "false"
      },
      {
         "key" : "REVISION",
         "value" : "20877f1b168b6ff27fc58aeef40756572c562d47"
      }
   ],
   "exporting" : false,
   "id" : "76ed4522a1ab2834eabd4b4a658b4b44874530157719fbb33cef005e241a13cc",
   "image" : "haugene/transmission-openvpn:latest",
   "is_ddsm" : false,
   "is_package" : false,
   "links" : [],
   "memory_limit" : 0,
   "name" : "haugene-transmission-openvpn1",
   "network" : [
      {
         "driver" : "bridge",
         "name" : "bridge"
      }
   ],
   "network_mode" : "bridge",
   "port_bindings" : [
      {
         "container_port" : 8118,
         "host_port" : 30000,
         "type" : "tcp"
      },
      {
         "container_port" : 9091,
         "host_port" : 30001,
         "type" : "tcp"
      }
   ],
   "privileged" : false,
   "shortcut" : {
      "enable_shortcut" : false,
      "enable_status_page" : false,
      "enable_web_page" : false,
      "web_page_url" : ""
   },
   "use_host_network" : false,
   "volume_bindings" : [
      {
         "host_volume_file" : "/docker/transmission-openvpn/custom",
         "mount_point" : "/etc/openvpn/custom",
         "type" : "rw"
      },
      {
         "host_volume_file" : "/docker/transmission-openvpn/resolv.conf",
         "mount_point" : "/etc/resolv.conf",
         "type" : "ro"
      },
      {
         "host_volume_file" : "/docker/transmission-openvpn/Watch",
         "mount_point" : "/data/watch",
         "type" : "rw"
      },
      {
         "host_volume_file" : "/docker/transmission-openvpn/Incomplete",
         "mount_point" : "/data/incomplete",
         "type" : "rw"
      },
      {
         "host_volume_file" : "/docker/transmission-openvpn/Downloads",
         "mount_point" : "/data/completed",
         "type" : "rw"
      }
   ]
}

**Comment by pkishino** [comment, marked as answer]
synology?

**Reply by cruncher007** [reply]
> synology?

Yes. Is this a known issue?

**Reply by pkishino** [reply]
yes, suggest checking here:
https://github.com/haugene/docker-transmission-openvpn/issues/1839#issuecomment-955632293

**Reply by cruncher007** [reply]
thanks, worked wonders!
